docs/src/_reference/command-line-interface.md

```diff
@@ -730,13 +729,64 @@ meltano job list <job_name> --format=json
 meltano job remove <job_name>
 ```
 
+
 ### Examples
 
+##### Tasks and Blocks
+A task in a job consists of one or more _blocks_. A block is simply a set of related plugins that will be executed serially.
+For example, you can create a job that consists of a single tap and target:
+```bash
+meltano job add tap-gitlab-to-target-postgres --tasks "tap-gitlab target-postgres"
+```
+This would add the following to your `meltano.yml`:
+```yaml
```

Comment:
Ditto comment above also for code blocks. I think each code block or paragraph should have a blank line between.